src/clarity-angular/tooltips/_tooltips.clarity.scss

```diff
@@ -186,6 +186,7 @@ $clr-tooltip-adjusted-margin: $margin-value + $arrow-width * 2 !default;
     .tooltip {
         & > .clr-icon{
```

Comment:
`.clr-icon` isn't a class that is used with the Clarity Icon library. It's legacy from before our custom element based Icon framework.

Also, you probably want the pointer-events to be on the `svg` element inside the Clarity Icon.

So this line should be more like...

```
.tooltip {
  & > .clr-icon {
    margin-right: 0;
  }

  clr-icon > svg {
    pointer-events: none;
  }
}
```